src/NumericalAlgorithms/LinearOperators/PartialDerivatives.cpp

```diff
@@ -293,16 +294,20 @@ GENERATE_INSTANTIATIONS(INSTANTIATION, (1, 2, 3))
 
 #undef INSTANTIATION
 
-#define INSTANTIATION(r, data)                                                \
+#define INSTANTIATE_SCALAR(r, data)                                           \
   template void partial_derivative<Symmetry<>, index_list<>>(                 \
-      gsl::not_null<tnsr::i<DataVector, GET_DIM(data), GET_FRAME(data)>*> du, \
+      const gsl::not_null<                                                    \
```

Comment:
just an FYI, you don't need the `const` in the explicit instantiations. I wouldn't put the effort in to remove them though :)

Reply:
hehe looks like this triggered both of us :D

Reply:
That's really funny you both reacted in the same way to the same thing, which is exactly why I *am* changing it lol going to do The Right Thing (TM)